demo/res/demo/app.json

```diff
@@ -17,6 +17,6 @@
     ],
     "runtime": {
         "arguments": "--enable-aggressive-domstorage-flushing",
-        "version": "9.61.38.40"
+        "version": "stable"
```

Comment:
Why do we want demo app running against 'stable' rather than a fixed version? I don't think there's anywhere else we're targeting stable